src/abbey/functions/has_attribute.rego

```diff
@@ -0,0 +1,7 @@
+package abbey.functions
+
+import future.keywords.if
+
+has_attribute(name, value) := true if {
```

Comment:
Please add code comment to document what this function is supposed to be doing. Same for other functions.